code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -73,6 +73,10 @@
 	if(!can_xeno_slash(X))
 		return FALSE
 
+	if(isanimal(src) && (stat == DEAD))
```

Comment:
bad. In general, istype(src) are verboten. 

Here you must overide attack_alien_harm for animal mobs.